.travis.yml

```diff
@@ -1,25 +1,13 @@
 language: php
 php:
-  - 5.6
-  - 7.0
-  - 7.1
-  - 7.2
+  - 7.4
```

Comment:
7.3 is still supported

Reply:
Let's keep supporting the old versions of PHP.